install/kubernetes/helm/istio/values.yaml

```diff
@@ -1,38 +1,38 @@
+# Use --set or additional values.yaml file to configure settings.
+# This file no longer uses sed, updateVersions.sh or istio.VERSIONS
+# TODO: evaluate if we need individual overrides for each component version, istio
+# is not typically tested with a mix of versions. Only supported case is version upgrade.
+
+# Common settings.
 global:
+  # Default tag for Istio images.
+  hub: istio
+
+  # Default tag for Istio images.
+  # The default is the latest released version, will be updated after
+  # each release.
+  tag: 0.5.1
   proxy:
-    repository: {PROXY_HUB}/{PROXY_IMAGE}
-    tag: {PROXY_TAG}
-  proxyInit:
-    repository: {PROXY_HUB}/proxy_init
-    tag: {PROXY_TAG}
-  sidecarInjector:
-    repository: {PILOT_HUB}/sidecar_injector
-    tag: {PILOT_TAG}
-  mixer:
-    repository: {MIXER_HUB}/mixer
-    tag: {MIXER_TAG}
-  pilot:
-    repository: {PILOT_HUB}/pilot
-    tag: {PILOT_TAG}
-  security:
-    repository: {CA_HUB}/istio-ca
-    tag: {CA_TAG}
+    image: proxy
+
+  # imagePullPolicy is applied to istio control plane components.
+  imagePullPolicy: IfNotPresent
+
   # Not recommended for user to configure this. Hyperkube image to use when creating custom resources
   hyperkube:
-    repository: {HYPERKUBE_HUB}
-    tag: {HYPERKUBE_TAG}
-  # addons
-  grafana:
-    repository: {MIXER_HUB}/grafana
-    tag: {MIXER_TAG}
-  servicegraph:
-    repository: {MIXER_HUB}/servicegraph
-    tag: {MIXER_TAG}
+    repository: quay.io/coreos/hyperkube
+    tag: v1.7.6_coreos.0
+
+  # Install istio CA. Needed for control plane auth, even if automatic mtls between pods is
+  # disabled. If false, you must also disable control plane security
+  securityEnabled: true
 
-  # create RBAC resources
+  # Default mtls policy. If true, mtls between services will be enabled by default.
+  mtlsDefault: false
+
+  # create RBAC resources. Must be set for any cluster configured with rbac.
   rbacEnabled: true
-  # mutual TLS authentication
```

Comment:
for rbacEnabled... I'd assume it is not valid when security is enabled?   and mixer is enabled today...

Reply:
AFAIK RBAC is required if the cluster has rbac ( and many have it on by default ).
RBAC only registers the cluster permissions so pilot can read stuff.